test/functional/apps/management/_import_objects.ts

```diff
@@ -49,7 +49,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
       it('should import saved objects', async function () {
         await PageObjects.savedObjects.importFile(
-          path.join(__dirname, 'exports', '_import_objects.ndjson')
+          path.join(__dirname, 'exports', '_import_objects.ndjson'),
+          { createNewCopies: false, overwrite: true }
         );
```

Comment:
On the server side, `createNewCopies` is enabled by default and `overwrite` is disabled by default. However, on the client side, `overwrite` is enabled by default too -- it is just ignored if `createNewCopies` is enabled too.

These tests were written before `createNewCopies` was introduced, so I'm leaving them in place with their expected options.